images/tidb-operator-e2e/tidb-cluster-values.yaml

```diff
@@ -224,7 +224,7 @@ fullbackup:
   binlogImage: pingcap/tidb-binlog:v3.0.0-rc.1
   binlogImagePullPolicy: IfNotPresent
   # https://github.com/tennix/tidb-cloud-backup
-  mydumperImage: pingcap/tidb-cloud-backup:latest
+  mydumperImage: pingcap/tidb-cloud-backup:20190610
```

Comment:
It looks like this change was made for testing and shouldn't be committed? Perhaps we need to improve our e2e testing workflow.

Reply:
Okay to revert the changes to e2e testing. Just thought it'd make more sense to test with the same version of tidb-cloud-backup as the main code.

Reply:
Another thing is - is this tag `20190610` appropriate to be used in tidb-operator? The only reason why I replaced this is because the `latest` tag is outdated, and doesn't support Amazon S3.

Reply:
the `20190610` is the latest of `tidb-cloud-backup` image. use it instead of `latest`.

Reply:
@shinnosuke-okada Do we have to update the [tidb-cloud-backup](https://github.com/pingcap/tidb-cloud-backup) code to support S3 backup/restore?

Reply:
@tennix Yes, the `latest` tag is outdated, and cannot be used to upload to/download from Amazon S3.

Reply:
@tennix My bad! Just realized that I misread your message here last time. No need to change the *code*. The latest code is good enough to support S3 backup/restore, and it's available via `20190610`. If you plan to rebuild the `latest` tag or some other tag, we can use that as well.